Cargo.toml

```diff
@@ -162,10 +162,10 @@ lto = "thin"
 overflow-checks = false
 codegen-units = 1       # Reduce number of codegen units to increase optimizations.
```

Comment:
Maybe we can remove this. `codegen-units = 1` increase the compile time a lot.

Reply:
removed